SimTracker/SiPixelDigitizer/plugins/SiPixelDigitizerAlgorithm.cc

```diff
@@ -1588,6 +1647,33 @@ void SiPixelDigitizerAlgorithm::pixel_inefficiency(const PixelEfficiencies& eff,
   std::vector<double> pixelEfficiencyROCStdPixels(16,1);
   std::vector<double> pixelEfficiencyROCBigPixels(16,1);
   
+  auto pIndexConverter = PixelIndices(numColumns,numRows);
+  
+  std::vector<int> badRocsFromFEDChannels(16,0);  
+  if (eff.PixelFEDChannelCollection_ != nullptr){  
+    PixelFEDChannelCollection::const_iterator it = eff.PixelFEDChannelCollection_->find(detID);
+    
+    if (it != eff.PixelFEDChannelCollection_->end()){      
+      for(const auto& ch: *it) {	
+	for (unsigned int i_roc = ch.roc_first; i_roc <= ch.roc_last; ++i_roc){
+	  std::vector<CablingPathToDetUnit> path = map_->pathToDetUnit(detID);
```

Comment:
This line doesn't seem to depend on either `i_roc` or `ch`, could it be moved outside of the two for-loops? (to minimize the times this vector gets copied; even better would be avoid the vector copy altogether but that's beyond this PR)